src/lucky/exceptions.cr

```diff
@@ -22,6 +23,10 @@ module Lucky
       def message : String?
         "Required param \"#{param_name}\" with value \"#{param_value}\" couldn't be parsed to a \"#{param_type}\""
       end
+
+      def default_response_status_code
```

Comment:
Actually maybe “default” isn’t necessary. And it might be good to be clear it is an HTTP status code. So what do you think about calling it http_error_code?